WebRoot/css/zm.css

```diff
@@ -7281,30 +7281,30 @@ input[type="radio"] + label {
 	@WhiteBlackListViewItemsNumberUsed@
 }
 
-.ZOptionsSectionMain .ZmBlockMessage {
+.ZOptionsSectionMain .ZmPrefMailBlockMessage {
 	@PrefMailBlockMessageCell@
 }
 
-.ZOptionsLabelTop.ZmComposeLabel {
+.ZOptionsLabelTop.ZmPrefMailComposeLabel {
 	@PrefMailComposeLabel@
 }
 
-.ZmPrefMailCompose .ZOptionsLabelNarrow {
+.ZmPrefMailComposeContent .ZOptionsLabelNarrow {
 	@PrefMailComposeRowLabel@
 }
 
-.ZmPrefMailCompose .ZOptionsField {
+.ZmPrefMailComposeContent .ZOptionsField {
 	@PrefMailComposeRowValueWrapper@
 }
 
-.ZmPrefMailCompose .ZOptionsField .ZSelect > table {
+.ZmPrefMailComposeContent .ZOptionsField .ZSelect > table {
 	@PrefMailComposeRowSelectWidget@
 }
 
-.ZmPrefMailCompose .ZWidgetTable {
+.ZmPrefMailComposeContent .ZWidgetTable {
```

Comment:
.ZWidgetTable is a generic classes for a lot of widgets, do we want to style all those ? The property names suggests that its only relevant for ColorPicker.